src/features/search/redux/middleware.ts

```diff
@@ -0,0 +1,24 @@
+import { Middleware } from 'redux'
+import { UPDATE_TICKER } from '../redux'
+import {  } from 'redux/actions'
+import { AppState, MiddlewareDependencies } from 'models'
+import { stockChange } from 'redux/actions/stockChange'
+
+export const searchMiddleware = ({socket}: MiddlewareDependencies): Middleware<AppState> => {
```

Comment:
Does this work? The typings I've got here say that store type is the second type parameter to `Middleware`

Reply:
Yes I saw this and figured I'd just have to put an empty object first but it seems to work regardless.

I'll change to empty object preceeding state but will also look into why this might work.

Reply:
The first type is `DispatchExt`, I believe. Not entirely sure what that is, but the state type is definitely the 2nd parameter.

Reply:
looking at the .d.ts file the first type parameter isn't even used ¯\\\_(ツ)\_/¯ maybe it's there just in case they need it later